pkgs/by-name/x1/x11idle/package.nix

```diff
@@ -6,12 +6,12 @@
   libX11,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "9.2.4";
   pname = "x11idle-org";
 
   src = fetchurl {
-    url = "https://code.orgmode.org/bzg/org-mode/raw/release_${version}/contrib/scripts/x11idle.c";
+    url = "https://code.orgmode.org/bzg/org-mode/raw/release_${finalAttrs.version}/contrib/scripts/x11idle.c";
     sha256 = "0fc5g57xd6bmghyl214gcff0ni3idv33i3gkr339kgn1mdjljv5g";
```

Comment:
non-blocking nit: `sha256` -> `hash` as part of the refactor commit

Reply:
changing the hash causes the package to not bulid anymore

Reply:
```console
$ curl https://code.orgmode.org/bzg/org-mode/raw/release_9.2.4/contrib/scripts/x11idle.c
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.22.1</center>
</body>
</html>
```

That's because the repository has been moved to https://git.savannah.gnu.org/cgit/emacs/org-mode.git/ it seems.
This `fetchurl` is currently fetching an HTML document it seems

Reply:
`x11idle.c` is apparently [no longer](https://git.savannah.gnu.org/cgit/emacs/org-mode.git/log/?qt=grep&q=x11idle.c) in this repository, so this is probably why you cant fetch it.

Reply:
What should we do about this?

Reply:
@swflint any opinion on this? do you still need this package?